src/app/loadout-builder/ModPickerFooter.m.scss

```diff
@@ -4,7 +4,7 @@
   composes: flexRow from '../dim-ui/common.m.scss';
   align-items: center;
 
-  > * {
+  div {
```

Comment:
Without the descendent selector this'll match too much

Reply:
Done, I pushed it down to where I needed it so there is some separation between mods.